### ISSUE TITLE
[Feature Request]: Manual modifying create/update time, magic command to insert them into the note

### ISSUE THREAD

**Issue by shagru** [HappenApps/Quiver#375]
Love the Quiver App and it is getting better and better!

I would love to be able to have the option to manually modify the create/update time, since I'd like to keep the note in chronological order. When I am transferring notes taken elsewhere to Quiver, I would like to keep the original creation time. Right now I could hack it and modify it in the note json source. But it would be nice to have a easy way to do it. 

Also one step further, it would be nice if there is some macro or magic command that can be used to get the create/update time and insert it into the note. This is especially nice when exporting so that you can have a time information regarding the note.

Thanks again for the nice app!

**Comment by ylian** [comment]
Good idea. Will implement a way to change modified date and created date.
